### PR TITLE
Update teachers

### PR DESCRIPTION
**Pull request by Zubcz**

### CODE REVIEW

File: teachers/index.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@ První cvičení od 9:15::
 
 Druhé cvičení od 11:45::
   * xref:hroncmir#[Miro Hrončok] (garant předmětu)
-  * xref:hanusji8#[Jiří „Hany“ Hanuš]
+  * xref:borskjir#[Jiří „Zub“ Borský]
```
Comment (hroncok) [inline teachers/index.adoc]: Wrong class.

**Comment by Hanyman8** [comment]
I would like to help with the BI-3DT as well, reviewing, updating pages or what needs to be done. Can you move me to "Na přemětu se také podílejí a vypomáhají", please? What @hroncok thinks about that?

**Comment by Zubcz** [comment]
> I would like to help with the BI-3DT as well, reviewing, updating pages or what needs to be done. Can you move me to "Na přemětu se také podílejí a vypomáhají", please? What @hroncok thinks about that?

I've already change it. I think It' no problem. Your help is appreciated.

File: teachers/borskjir.adoc [hunk 2]
```diff
@@ -0,0 +1,6 @@
+= Jiří "Zub" Borský
```
Comment (hroncok) [inline teachers/borskjir.adoc]:
```suggestion
= Jiří „Zub“ Borský
```

**Pull request merged**